tests/bin/init-workflow.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash -e
 
-if [ "$BASE64_MATRIX" == "" ]
+if [ "$BASE64_OS" != "" ]
 then
-    MATRIX="{\"os\":[\"latest\"]}"
+    OS_VERSION=$(echo "$BASE64_OS" | base64 -d)
+elif [ "$BASE64_MATRIX" != "" ]
```

Comment:
Do we need this deprecated option still? Can we not simply replace the matrix with os?

Reply:
Not really. I just thought it a common practice to deprecate first if possible before removing something. But since these params are meant to be temporary, it might be OK to just replace it. Should I go ahead and replace it?

Reply:
OK, I went ahead and dropped `BASE64_MATRIX` completely.

Reply:
Yeah it is good practice in general for sure. I think in this case though it is fine to drop it as it is only us working with this parameter and we are certain that we have accounted for every use of it.